code/modules/clothing/neck/_neck.dm

```diff
@@ -5,7 +5,9 @@
 	slot_flags = ITEM_SLOT_NECK
 	strip_delay = 40
 	equip_delay_other = 40
-
+	///Vision flags granted by wearing this
+	var/trinket_flag = 0
+	var/trinket_lighting_alpha
```

Comment:
This variable name is a bit odd. What's the connection between trinket and and vision?

Reply:
i have to rework these either way, i shitcoded the trinket code at 3 am 5 days ago